Standards/scs-XXXX-v1-volume-type-standard.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Volume Type Standard
+type: Standard
+status: Draft
+track:  IaaS
+---
+
+## Introduction
+
+Volume Types are used to classify volumes and provide a basic decision for what kind of volume should be created. These volume types can sometimes be very backend-specific and it might be hard for a user to choose the most suitable volume type, if there is more than one type.
```

Comment:
Comparing this MR with https://github.com/SovereignCloudStack/standards/pull/453/ I prefer MR #453, as it's more precise.

Reply:
I changed it.